classes/core/PKPComponentRouter.inc.php

```diff
@@ -174,6 +174,10 @@ function &getRpcServiceEndpoint($request) {
 			// time.
 			$this->_rpcServiceEndpoint = $nullVar = null;
 
+			// Retrieve requested component operation
+			$op = $this->getRequestedOp($request);
```

Comment:
It's been a while since I looked at this, so forgive a potentially silly question -- but I think it's the `page` that component URLs lack, not the `op`, no?